Library/Homebrew/dev-cmd/linkage.rb

```diff
@@ -84,6 +88,21 @@ def display_normal_output
       display_items "Possible undeclared dependencies", @undeclared_deps
     end
 
+    def display_reverse_output
+      return if @reverse_links.empty?
+      puts "Links to:"
+      sorted = @reverse_links.sort
+      sorted.each do |dylib, files|
+        next if dylib == "/usr/lib/libSystem.B.dylib"
+        puts dylib
```

Comment:
I think this should be indented by two spaces (or alternatively the “Links to:” line completely dropped). Right now the indentation of the output looks just off.